packages/dune-rpc-lwt/dune-rpc-lwt.3.0.0/opam

```diff
@@ -9,10 +9,10 @@ doc: "https://dune.readthedocs.io/"
 bug-reports: "https://github.com/ocaml/dune/issues"
 depends: [
   "dune" {>= "3.0"}
-  "result"
+  "result" {>= "1.5"}
```

Comment:
@kit-ty-kate where did this constraint come from? I think I can actually just remove this dependency. We don't use the `result` package at all anymore.

Reply:
Oh never mind. We require it because of Lwt.

Reply:
This one was just a guess, given anything below 1.5 doesn’t have compatible types with the standard library (`Result.t != Stdlib.Result.t`)

Reply:
Note that it won’t be the case in Lwt 5.6.0: https://github.com/ocsigen/lwt/pull/925